modules/solid_mechanics/src/userobjects/CrackFrontDefinition.C

```diff
@@ -381,6 +382,13 @@ CrackFrontDefinition::orderCrackFrontNodes(std::set<dof_id_type> &nodes)
       last_node = _ordered_crack_front_nodes.back();
     }
   }
+  //Store the order crack front nodes into _crack_front_points
+  _crack_front_points.resize(_ordered_crack_front_nodes.size());
+  for (unsigned int nit = 0; nit != _ordered_crack_front_nodes.size(); ++nit)
+  {
+    Node & crack_front_node = _mesh.node(_ordered_crack_front_nodes[nit]);
```

Comment:
More of an FYI since it's probably not important for now. This code will likely fail with parallel mesh. However I doubt that the XFEM work in general will work with parallel mesh. This is important as we continue to expand the test platforms for the modules. These should be documented and restricted for those cases. You don't need to fix this now, just be aware.